testing/libarchive-qt/APKBUILD

```diff
@@ -0,0 +1,25 @@
+pkgname=libarchive-qt
+pkgver=1.0.2
+pkgrel=0
+pkgdesc="A Qt based archiving solution with libarchive backend"
+arch="all"
+url="https://gitlab.com/marcusbritanicus/libarchive-qt"
+license="LGPL-3.0"
+depends="xz libarchive zlib lzip bzip2"
```

Comment:
depends= can be totally removed.

below depends= add:

depends_dev="qt5-qtbase-dev xz-dev libarchive-dev zlib-dev bzip2-dev"

then modify makedepends to:

makedepends="$depends_dev lzip"

Reply:
Also this is why the packages that depend on libarchive-qt aren't building, depends_dev is the list of packages that the libarchive-qt-dev package will depend on, with your current configuration it will install `libarchive-qt-dev` but won't install the dependencies required.

Your packages then fail to compile because it is missing the `archive.h` header which is part of libarchive-dev which should be automatically installed when you install libarchive-qt-dev